firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -48,15 +48,18 @@ bool isValidIM21Id(String input) {
 }
 
 int toggle_debugging_publishes(String command){
-
   //default to invalid input
   int returnFlag = -1;
 
   //string.toInt() returns 0 if it fails, so can't distinguish between user 
   //entering 0 vs entering bad input. So convert to char and use ascii table
   const char* holder = command.c_str();
 
-  if(*(holder+1) != 0){
+  // if e, echo whether debug publishes are on
+  if (*holder == 'e') {
+    returnFlag = (int)stateMachineDebugFlag;
+  }
```

Comment:
What do you think about moving this to an `else if` after the `if(*(holder+1) != 0){`?

I think the reason for the `if(*(holder+1) != 0){` test going first is to speed up this function. It cuts out a bunch of invalid data in one go before checking for the now three different expected values (e, 0, and 1).

Reply:
That's a good point Theresa. I think the context we use the function in will not result in a lot of invalid inputs. But I agree it's a good practice so I've made the change in my latest commit